doc/design/libraries.md

```diff
@@ -54,9 +54,6 @@ bitcoin-wallet[bitcoin-wallet]-->libbitcoin_wallet_tool;
 libbitcoin_cli-->libbitcoin_common;
 libbitcoin_cli-->libbitcoin_util;
 
-libbitcoin_common-->libbitcoin_util;
-libbitcoin_common-->libbitcoin_consensus;
```

Comment:
I think this documents that a dependency is *allowed*, not *enforced*

Reply:
Sorry, I was just wrong. Dependencies are true.